backend/config/autoload/doctrine.local.test.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
```

Comment:
Dieser Inhalt ist bereits in "/backend/config/autoload/doctrine.global.test.php" vorhanden.
File wird von "backend/config/application.config.php" geladen, da ENV auf "test" gesetzt wird